Tasks/ManualIntervention/task.json

```diff
@@ -30,6 +30,14 @@
                 "rows": "10",
                 "maxLength": "500"
             }
+        },
+        {
+            "name": "emailRecipients",
+            "type": "identities",
```

Comment:
I am not sure how you the backend is processing this. You should be supporting IDN mail ids as well.